Here's crash report
```
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'skyblocker' at 'de.hysky.skyblocker.SkyblockerMod'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$0(FabricLoaderImpl.java:409)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:407)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.client.Minecraft.<init>(Minecraft.java:477)
	at knot//net.minecraft.client.main.Main.main(Main.java:232)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at com.modrinth.theseus.MinecraftLaunch.relaunch(MinecraftLaunch.java:61)
	at com.modrinth.theseus.MinecraftLaunch.main(MinecraftLaunch.java:28)
Caused by: java.lang.ExceptionInInitializerError
	at knot//de.hysky.skyblocker.skyblock.item.tooltip.TooltipManager.<clinit>(TooltipManager.java:52)
	at knot//de.hysky.skyblocker.SkyblockerMod.init(SkyblockerMod.java)
	at knot//de.hysky.skyblocker.SkyblockerMod.onInitializeClient(SkyblockerMod.java:55)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:405)
	... 10 more
Caused by: java.lang.IllegalArgumentException: Keys and values have different lengths (9, 8)
	at knot//it.unimi.dsi.fastutil.objects.Object2LongArrayMap.<init>(Object2LongArrayMap.java:62)
	at knot//de.hysky.skyblocker.skyblock.item.tooltip.adders.EssenceShopPrice.<clinit>(EssenceShopPrice.java:26)
	... 14 more
```

--- a/src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/EssenceShopPrice.java
+++ b/src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/EssenceShopPrice.java
@@ -22,8 +22,8 @@ import org.jspecify.annotations.Nullable;
 
 public class EssenceShopPrice extends SimpleTooltipAdder {
 	private static final Pattern ESSENCE_PATTERN = Pattern.compile("Cost (?<amount>[\\d,]+) (?<type>[A-Za-z]+) Essence");
-	private static final String[] ESSENCE_TYPES = {"WITHER", "SPIDER", "UNDEAD", "DRAGON", "GOLD", "DIAMOND", "ICE", "CRIMSON", "FOSSIL"};
-	private static final Object2LongArrayMap<String> ESSENCE_PRICES = new Object2LongArrayMap<>(ESSENCE_TYPES, new long[9]);
+	private static final String[] ESSENCE_TYPES = {"WITHER", "SPIDER", "UNDEAD", "DRAGON", "GOLD", "DIAMOND", "ICE", "CRIMSON", "FOREST", "FOSSIL"};
+	private static final Object2LongArrayMap<String> ESSENCE_PRICES = new Object2LongArrayMap<>(ESSENCE_TYPES, new long[10]);
 
 	public EssenceShopPrice(int priority) {
 		super("\\S+ Essence Shop", priority);

--- a/src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/EssenceShopPrice.java
+++ b/src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/EssenceShopPrice.java
@@ -23,7 +23,7 @@ import org.jspecify.annotations.Nullable;
 public class EssenceShopPrice extends SimpleTooltipAdder {
 	private static final Pattern ESSENCE_PATTERN = Pattern.compile("Cost (?<amount>[\\d,]+) (?<type>[A-Za-z]+) Essence");
 	private static final String[] ESSENCE_TYPES = {"WITHER", "SPIDER", "UNDEAD", "DRAGON", "GOLD", "DIAMOND", "ICE", "CRIMSON", "FOSSIL"};
-	private static final Object2LongArrayMap<String> ESSENCE_PRICES = new Object2LongArrayMap<>(ESSENCE_TYPES, new long[8]);
+	private static final Object2LongArrayMap<String> ESSENCE_PRICES = new Object2LongArrayMap<>(ESSENCE_TYPES, new long[9]);
 
 	public EssenceShopPrice(int priority) {
 		super("\\S+ Essence Shop", priority);

--- a/src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Ico.java
+++ b/src/main/java/de/hysky/skyblocker/skyblock/tabhud/util/Ico.java
@@ -113,4 +113,5 @@ public class Ico {
 	public static final FlexibleItemStack ROSE_BUSH = new FlexibleItemStack(Items.ROSE_BUSH);
 	public static final FlexibleItemStack GOLD_NUGGET = new FlexibleItemStack(Items.GOLD_NUGGET);
 	public static final FlexibleItemStack NAUTILUS_SHELL = new FlexibleItemStack(Items.NAUTILUS_SHELL);
+	public static final FlexibleItemStack FLOWERING_AZALEA_LEAVES = new FlexibleItemStack(Items.FLOWERING_AZALEA_LEAVES);
 }

--- a/src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/EssenceWidget.java
+++ b/src/main/java/de/hysky/skyblocker/skyblock/tabhud/widget/EssenceWidget.java
@@ -34,6 +34,7 @@ public class EssenceWidget extends TabHudWidget {
 				case String s when s.contains("diamond") -> this.addComponent(Elements.iconTextComponent(Ico.DIAMOND, line));
 				case String s when s.contains("ice") -> this.addComponent(Elements.iconTextComponent(Ico.ICE, line));
 				case String s when s.contains("crimson") -> this.addComponent(Elements.iconTextComponent(Ico.REDSTONE, line));
+				case String s when s.contains("forest") -> this.addComponent(Elements.iconTextComponent(Ico.FLOWERING_AZALEA_LEAVES, line));
 				case String s when s.contains("fossil") -> this.addComponent(Elements.iconTextComponent(Ico.NAUTILUS_SHELL, line));
 				default -> this.addComponent(new PlainTextElement(line));
 			}

